From f73655e20d3974067a64800109946adefefd715a Mon Sep 17 00:00:00 2001
From: ksjogo <jogo@kronberger-spiele.de>
Date: Fri, 14 Aug 2015 12:53:39 +0200
Subject: [AZURE] .user.ini

---
 .user.ini | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 .user.ini

diff --git a/.user.ini b/.user.ini
new file mode 100644
index 0000000..48681de
--- /dev/null
+++ b/.user.ini
@@ -0,0 +1,4 @@
+upload_max_filesize = 10M
+post_max_size = 10M
+max_execution_time = 240
+max_input_vars = 1500
-- 
2.7.0

