From da9ab266f8d3b326e16b856eb8e7f16902e62bf1 Mon Sep 17 00:00:00 2001
From: Johannes Goslar <johannes.goslar@dkd.de>
Date: Fri, 14 Aug 2015 12:53:39 +0200
Subject: [PATCH 6/6] .user.ini

---
 .user.ini | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 .user.ini

diff --git a/.user.ini b/.user.ini
new file mode 100644
index 0000000..fe0573a
--- /dev/null
+++ b/.user.ini
@@ -0,0 +1,8 @@
+upload_max_filesize = 10M
+post_max_size = 10M
+max_execution_time = 240
+max_input_vars = 1500
+wincache.maxfilesize = 4096
+wincache.reroute_enabled = On
+wincache.ocenabled = On
+opcache.enable = Off
-- 
1.9.4.msysgit.1

