From 2ff13d2286edae540e496cee3f3d146a3c3264d0 Mon Sep 17 00:00:00 2001
From: dkd-azure <dkd-azure>
Date: Tue, 25 Aug 2015 16:42:47 +0200
Subject: [PATCH] Fix Wincache on Local Machines


diff --git a/typo3/sysext/core/Classes/Service/OpcodeCacheService.php b/typo3/sysext/core/Classes/Service/OpcodeCacheService.php
index 714250b..1142a57 100644
--- a/typo3/sysext/core/Classes/Service/OpcodeCacheService.php
+++ b/typo3/sysext/core/Classes/Service/OpcodeCacheService.php
@@ -59,12 +59,7 @@ class OpcodeCacheService {
 				'canInvalidate' => TRUE, // wincache_refresh_if_changed()
 				'error' => FALSE,
 				'clearCallback' => function ($fileAbsPath) {
-					if ($fileAbsPath !== NULL) {
-						wincache_refresh_if_changed(array($fileAbsPath));
-					} else {
-						// No argument means refreshing all.
-						wincache_refresh_if_changed();
-					}
+					wincache_refresh_if_changed();
 				}
 			),
 
-- 
1.9.4.msysgit.1

