From cd56bb67dd8eed2e209d9765988c867a57f1b2a7 Mon Sep 17 00:00:00 2001
From: ksjogo <jogo@kronberger-spiele.de>
Date: Fri, 14 Aug 2015 12:53:39 +0200
Subject: [PATCH] .user.ini

---
 .user.ini | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 .user.ini

diff --git a/.user.ini b/.user.ini
new file mode 100644
index 0000000..0797a5b
--- /dev/null
+++ b/.user.ini
@@ -0,0 +1,8 @@
+upload_max_filesize = 10M
+post_max_size = 10M
+max_execution_time = 240
+max_input_vars = 1500
+wincache.maxfilesize = 4096
+wincache.reroute_enabled = On
+wincache.ocenabled = On
+opcache.enable = Off
-- 
2.5.0

